test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -163,5 +207,36 @@ private void DeleteProducts(int first_id, int last_id)
                 id += 1;
             }
         }
+
+        private async Task<string> AssertTriggerListenerError(string functionName)
+        {
+            string errorMessage = null;
+            var tcs = new TaskCompletionSource<bool>();
+
+            void OutputHandler(object sender, DataReceivedEventArgs e)
+            {
+                if (errorMessage == null && e.Data?.Contains("Failed to start SQL trigger listener") == true)
+                {
+                    string exceptionPrefix = "Exception: System.InvalidOperationException: ";
+                    int index = e.Data.IndexOf(exceptionPrefix, StringComparison.Ordinal);
+                    Assert.NotEqual(-1, index);
+                    errorMessage = e.Data[(index + exceptionPrefix.Length)..];
+                    tcs.SetResult(true);
+                }
+            };
+
+            this.StartFunctionHost(functionName, Common.SupportedLanguages.CSharp, false, OutputHandler);
```

Comment:
Make sure to call out in the doc for this that it's only running for C# at the moment.

Reply:
@Charles-Gagnon, do you mean in the summary comment of this class?

Reply:
Correct (or `remark`. Or honestly even just as a comment in the method, whatever you like best. I just want it called out somewhere so people looking at this down the line know it was done on purpose)